packages/core/src/node/initRsbuild.ts

```diff
@@ -156,7 +156,10 @@ async function createInternalBuildConfig(
                 config: {
                   ...tailwindConfig,
```

Comment:
The config in `packages/theme-default/tailwind.config.ts` should be included as well, which defines some neccessary theme tokens.

Reply:
How do you think about it @xc2

Reply:
yea it should apply theme-default's tw config as it's mostly for theme-default's source. 

will back by tonight.

Reply:
fix by the second commit - i moved `DOC_DEBUG`-related codes to a standalone rspress plugin in `theme-default`

Reply:
Awesome!